src/main/java/TestFile.java

```diff
@@ -13,22 +13,22 @@ private static void incorrectAssignmentInIfCondition() {
     }
 
     private static void incorrectComparingToItself() {
-        int x = 3423421;
-        if (x == x) {
+        int x = 3421;
+        if (x <= x) {
             // whatever
         }
     }
 
     private static void anotherVariationOfIncorrectComparingToItself() {
         int x = 567;
-        if (x <= x) {
+        if (x == x) {
             // whatever
         }
     }
 
     private static void onMoreVariationOfIncorrectComparingToItself() {
         int x = 23422;
-        if (x == x) {
```

Comment:
[Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.